browser/net/brave_ad_block_tp_network_delegate_helper.cc

```diff
@@ -325,6 +325,11 @@ int OnBeforeURLRequest_AdBlockTPPreWork(const ResponseCallback& next_callback,
     return net::OK;
   }
 
+  if (!ctx->request_url.SchemeIsHTTPOrHTTPS() &&
```

Comment:
i would move this closer to the checks in the beginning, just to have a logical storyline